tests/models/test_llama_cpp.py

```diff
@@ -42,69 +61,93 @@ def test_repeat_calls():
     a.append(lm['test'])
     assert a[-1] == a[0]
```

Comment:
just noticing this now, but shouldn't we set sampling params explicitly, e.g. temp=0, before testing that 10 tokens are exactly identical? either that or we should explicitly specify that this is a caching test